cmd/nodecmd/create_aws.go

```diff
@@ -355,6 +375,29 @@ func AddMonitoringSecurityGroupRule(ec2Svc map[string]*awsAPI.AwsCloud, monitori
 	return nil
 }
 
+func grantAccessToPublicIPViaSecurityGroup(ec2Svc *awsAPI.AwsCloud, publicIP, securityGroupName, region string) error {
```

Comment:
this is exactly the same func as `AddMonitoringSecurityGroupRule` why not to make it work for both cases